packages/devtools_app/lib/src/service_extensions.dart

```diff
@@ -172,10 +216,17 @@ final slowAnimations = ToggleableServiceExtensionDescription<num>._(
   icon: createImageIcon('icons/history@2x.png'),
   enabledValue: 5.0,
   disabledValue: 1.0,
-  enabledTooltip: 'Disable Slow Animations',
-  disabledTooltip: 'Enable Slow Animations',
   gaScreenName: analytics_constants.inspector,
   gaItem: analytics_constants.slowAnimation,
+  tooltipBuilder: (bool isSelected, Widget child) {
```

Comment:
based on the examples seen, why not add an optional `url` parameter to ToggleableServiceExtensionDescription instead of 
adding a tooltipBuilder? That would reduce how verbose the code is and would keep a cleaner abstraction between business logic and widget layers.
I'm ok with also adding a tooltipBuilder option if there are some future use cases that are more complex than just adding a url parameter. Even in that case, it could make sense to keep the `enabledTooltip`, `disabledTooltip`, and `docUrl`  for the simple cases.